code-challenges.js

```diff
@@ -17,26 +17,75 @@ const fruit4 = "kiwi"
 
 // --------------------1) Create a function that takes in any given number and determines if the number is below boiling point, at boiling point, or above boiling point. Boiling point is 212 degrees Fahrenheit. Use the test variables provided below. Expected output: "42 is below boiling point", "350 is above boiling point", "212 is at boiling point"
 
+//These variables set the temperatures to be used as the function arguments
 const temp1 = 42
 const temp2 = 350
 const temp3 = 212
 
+//Beginning of the function
+const boilingPoint = (temp) => {
+//First conditional statement returns the necessary string if the temperature is at the boiling point
+  if(temp == 212){
+    return temp + " is at boiling point"
```

Comment:
Logic looks great on this problem!  Often times it is best practice to use strict equality in conditionals rather than loose equality along with using string interpolation instead of string concatenation.